Index: www/js/controller/ReadviewViewController.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/www/js/controller/ReadviewViewController.js b/www/js/controller/ReadviewViewController.js
--- a/www/js/controller/ReadviewViewController.js	
+++ b/www/js/controller/ReadviewViewController.js	
@@ -23,7 +23,10 @@
             this.bindElement("mediaReadviewTemplate",{item: mediaItem},this.root).viewProxy;
 
         this.viewProxy.bindAction("deleteItem",(() => {
-            mediaItem.delete().then(() => {
+            // mediaItem.delete().then(() => {
+            //     this.previousView({deletedItem:mediaItem});
+            // })
+            this.deleteItem(mediaItem).then(() => {
                 this.previousView({deletedItem:mediaItem});
             })
         }));
@@ -75,7 +78,7 @@
         // TODO: check from which view, and possibly with which status, we are returning, and handle returnValue accordingly
     }
 
-    deleteItem(item) {
+   async deleteItem(item) {
         this.showDialog("mediaItemDeleteDialog",{
             item: item,
             actionBindings: {
Index: www/js/MyApplication.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/www/js/MyApplication.js b/www/js/MyApplication.js
--- a/www/js/MyApplication.js	
+++ b/www/js/MyApplication.js	
@@ -44,7 +44,6 @@
         // activate the local crud operations
         //this.initialiseCRUD(this.CRUDOPS.REMOTE,EntityManager);
 
-
         // TODO: do any further application specific initialisations here
 
         // THIS MUST NOT BE FORGOTTEN: initialise the entity manager!
